CHANGELOG.md

```diff
@@ -5,6 +5,44 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+ * [`changed`] `base_config.inc` to work with I2C and SHDLC at the same time.
+               Changed HAL implemantation locations and moved the UART one from
+               the Makefile of the test in the sensor repo to here. Also changed
+               the way how mux sources are included so they are only needed by i2c
+               tests.
+ * [`added`] SHDLC implementation. This implementation was immigrated from the
```

Comment:
Nitpick: Usually we align the whitespace of the text blocks, so add more spaces after the shorter tags like `[added]`.

Reply:
Okay I will correct that in the release (which I will do right after that one is merged) if that's okay?